front/pages/w/[wId]/members/index.tsx

```diff
@@ -253,18 +245,19 @@ export default function WorkspaceAdmin({
                     : `member-${item.id}`
                 }
                 className="transition-color flex cursor-pointer items-center justify-center gap-3 border-t border-structure-200 py-2 text-xs duration-200 hover:bg-action-100 sm:text-sm"
-                onMouseEnter={() => {
-                  if (isInvitation(item)) setInvitationToRevoke(item);
-                  else setChangeRoleMember(item);
-                }}
                 onClick={() => {
-                  if (isInvitation(item)) setRevokeInvitationModalOpen(true);
-                  else setChangeRoleModalOpen(true);
+                  if (isInvitation(item)) setInvitationToRevoke(item);
+                  else setChangeRoleMemberId(item.id);
```

Comment:
Why setting the id and then when we call `<ChangeMemberModal` line 214 we loop on members to send the full member. Why not keeping the member on the state? It would avoid the .find

Reply:
discussed irl, to get the changemember to mutate when members is mutated